resources/execute_testing.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Install edX app on emulator 
+APK=$APK_PATH/"*.apk"
+echo $APK
+$ANDROID_HOME/platform-tools/adb install $APK
+sleep 1m
+
+# Verify if edX app is installed successfully
+$ANDROID_HOME/platform-tools/adb shell pm list packages | grep org.edx.mobile
+if [ $? == 0 ]; then
+   echo "edX app is installed successfully"
+   #  ==============WIP==========
+   # Install dependencies 
+   exit 0
+else
+   echo "edX app is installed successfully"
```

Comment:
^ This should be an error message. It might be a better idea to do the following (rather than wrapping the install dependencies in the if block):
* check if installation failed
* if so print error message and exit 1
* carry on with the script, outside of the if block

Reply:
done